skbeam/core/roi.py

```diff
@@ -42,6 +42,7 @@
 
 import collections
 import scipy.ndimage.measurements as ndim
+from skimage.draw import line, polygon
```

Comment:
`flake8` tests are failing because you have imported `polygon` but it is not used. Remove it and tests will pass